dpu-cni/pkgs/cniserver/cniserver.go

```diff
@@ -247,14 +248,23 @@ func (s *Server) handleCNIRequest(r *http.Request) ([]byte, error) {
 		return nil, err
 	}
 
-	result, err := s.processRequest(&cniRq)
+	req, err := cniRequestToPodRequest(&cniRq)
 	if err != nil {
 		return nil, err
 	}
+	defer req.Cancel()
 
-	response := &cnitypes.Response{}
-	response.Result = result
+	var result *cni100.Result = nil
+	if req.Command == cnitypes.CNIAdd {
+		result, err = s.addHandler(req)
```

Comment:
Can we rename "addHandler" to  "cniCmdAddHandler"... "addHandler" and "delHandler" has different meanings on its own.

Reply:
+1